src/pages/Introduce/Introduce.js

```diff
@@ -0,0 +1,38 @@
+import React from "react"
+import "./Introduce.scss";
+import { Link, useNavigate } from "react-router-dom";
+
+
+function Introduce (){
+    const navigate = useNavigate();
+    
+    const handleButton = () => {
+        navigate('./product');
+    }
+
+    const handleInput = (e) => {
```

Comment:
```suggestion
    const handleInput = () => {
```
매개변수 자리에 불필요한 값이 들어와있습니다
사용하지 않는 값이라면 지워주세요!